### PR TITLE
Fix for RRFS and RTMA_dev1 (RRFS-based) skewT plots (hydrometeors and longitudes)

### PR DESCRIPTION
**Pull request by cshartsough**
This set of changes is to correct problems with  RRFS and  RRFS-based RTMA_dev1 skewT  plotting.  Two issues are corrected
- longitudes were not consistent for -180 to +180 versus 0 to 360.  Checks were added to adjust either way.
- The skewT  code crashed if any of the  hydrometeor fields were not in the grib file,  and cloud water is not currently in the files.  The code was set to skip over missing fields and continue.

passed pylint, pytest needs to be checked (my own local pytest has a big).

sample plots:
image1
image2
999_74001_skewt_f000

### CODE REVIEW

**Review by Brian-Jamison — APPROVED** [review]
LGTM!

File: adb_graphics/datahandler/gribdata.py [hunk 1]
```diff
@@ -852,10 +854,12 @@ def __init__(self, ds, loc, short_name, **kwargs):
...
         self.site_name = loc[37:].rstrip()
 
-        # Convert the string to a number. Longitude should be negative for all
+        # Convert the string to a number. Longitude should be positive for all
         # these sites.
         self.site_lat = float(lat)
-        self.site_lon = -float(lon)
+        self.site_lon = -float(lon) # lons are -180 but without minus sign in input file
+        if self.site_lon < 0:
+            self.site_lon = self.site_lon + 360.0
```
Comment (christinaholtNOAA) [inline adb_graphics/datahandler/gribdata.py:862]: I'm still a little lost with this comment (the original wasn't very clear either).

Let's make sure I'm getting this right. The input files (e.g., conus_raobs.txt) have a positive value for longitude. Here, it was being made negative to be consistent with maybe the specification of the grid in loaded by gribdata.py. That's a bad assumption to make for ALL our models (the rule was broken for RRFS NA), so now, we are ensuring that all data is converted to positive longitude in gribdata.py, and similarly converting profile locations to positive longitudes here, too.

Comment (cshartsough) [reply adb_graphics/datahandler/gribdata.py:862]: I think my comment in the code got corrupted.

The conus_raobs file uses -180 to 180, but leaves off the minus sign, i.e., the values are  in degrees West.   So, first we need to add the minus sign to convert the longitude to deg East, and then need to adjust to the 0 to 360 system.

**Review by christinaholtNOAA — APPROVED** [review]

**Review by christinaholtNOAA — APPROVED** [review]
The updated comment is much clearer! Thanks!

**Pull request merged**